pallets/avn-proxy/src/tests/proxy_tests_no_fees.rs

```diff
@@ -86,7 +86,7 @@ mod proxy_without_fees {
 
                 assert_eq!(result, expected_eth_address[..]);
 
-                let public_key_bytes: [u8; 32] = hex!("6fc436cd5273e4ecfd09079c982e2824825d5903cb2cf6cfd02586fbf6faa9a3");
+                let public_key_bytes: [u8; 32] = hex!("844da93c328fc981b1c507a926bb64055f01ec6a7b4d1ab9dc79e086655d368d");
```

Comment:
Can we use a const to store this?
Can we avoid using a raw public key instead of deriving it?

Reply:
In this test we're confirming the derivation in the function is correct, so we do need to hard-code the expected value. 

In the other test (`call_is_proxied_with_good_parameters_ecdsa`) however we rely on the correct key being derived by the function, so for that we accept whatever account gets generated. 

Between the two though we have both avenues covered.